lib/auth/auth.go

```diff
@@ -472,7 +472,7 @@ var (
 )
 
 // LoginHook is a function that will be called on login.
```

Comment:
At what stage of login is this supposed to be called? Does it matter? Could we document a bit more?

Reply:
So long as it happens sometime after success it doesn't really matter. Will do!